### PR TITLE
Fix args to work with contest system

### PR DESCRIPTION
**Pull request by tim1504**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -12,7 +12,7 @@ use logic::OwnLogic;
 #[derive(Parser, Debug)]
 struct Args {
     /// The game server's host address.
-    #[clap(long, default_value = "localhost")]
+    #[clap(short, long, default_value = "localhost")]
```
Comment (fwcd) [inline src/main.rs:15]: IIRC this was to prevent ambiguities with `-h`/`--help`. How does clap handle this?

Comment (tim1504) [reply src/main.rs:15]: ```-h``` for help is overwritten:
```
socha-client-2024 -h
error: a value is required for '--host <HOST>' but none was supplied

For more information, try '--help'. 
```

Comment (fwcd) [reply src/main.rs:15]: Ah okay, thanks for clarifying!

**Comment by fwcd** [comment]
Thanks!

**Pull request merged**